packages/ffe-icons-react/src/build-jsx-components.js

```diff
@@ -28,21 +36,18 @@ const createSvgMap = () => {
  * */
 const toJsx = svgString => {
     const $ = cheerio.load(svgString, {
-        xmlMode: true
+        xmlMode: true,
     });
     const svg = $('svg');
     // React does not support namespace definitions
     svg.attr('xmlns', null);
     svg.attr('xmlns:svg', null);
 
-    svg.attr('height', 150);
-    svg.attr('width', 150);
```

Comment:
Looks like the commit-hook went over the formatting, but the only real change is the removal of these lines.